src/ScreenshotManager.vala

```diff
@@ -23,6 +23,7 @@ namespace Gala {
     [DBus (name="org.gnome.Shell.Screenshot")]
     public class ScreenshotManager : Object {
         private WindowManager wm;
+        private NotificationsManager notifications_manager;
```

Comment:
Do we use it here anywhere?

Reply:
This is required for https://github.com/elementary/gala/pull/2241. But I removed it for now